src/components/Header/Header.css

```
@@ -0,0 +1,35 @@
+.language-switch {
+  cursor: pointer;
+  border-radius: 999px;
+  background-color: var(--white);
+  border: 1px solid var(--black);
+  padding: 8px 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 8px;
+}
+
+.language-switch:hover {
+  background-color: rgba(25, 13, 3, 0.1);
+}
+
+.logo-img {
+  height: 53px;
+}
+
+@media (max-width: 410px) {
+  .disappearing-text {
+    display: none;
+  }
+
+  .language-switch {
+    padding: 8px;
+  }
+}
+
+@media (max-width: 1024px) {
```

If you make it 1 pixel smaller, it will be perfect)

```suggestion
@media (max-width: 1023px) {
```

https://github.com/user-attachments/assets/58d0750e-5fd3-4f96-a53c-45acd9a8f2d2